website/source/guides/upgrade/index.html.md

```diff
@@ -8,30 +8,42 @@ description: |-
 
 # Upgrading
 
-This page documents how to upgrade Nomad when a new version is released.
-
-~> **Upgrade Warning!** Both Nomad Clients and Servers are meant to be
-long-running processes that maintain communication with each other. Nomad
-Servers maintain quorum with other Servers and Clients are in constant
-communication with Servers. As such, care should be taken to properly
-upgrade Nomad to ensure minimal service disruption. Unsafe upgrades can
-cause a service outage.
-
-~> **Downgrade Warning!** We currently do not support safely downgrading
-Nomad servers. This is due to the fact that the data directory
-contains transaction logs which may not safely apply after a downgrade. To downgrade
-a Nomad server, its data directory must be wiped out. In general, Client downgrades are
-safe to do, however we recommend checking the [upgrade specific](/guides/upgrade/upgrade-specific.html)
-page which will highlight versions where a client downgrade is not supported.
+Nomad is designed to be flexible and resilient when upgrading from one Nomad
+version to the next. Upgrades should cause neither a Nomad nor a service
+outage. However, there are some restrictions to be aware of before upgrading:
 
-## Upgrade Process
+ * Nomad strives to be backward compatible for at least 1 point release, so
+   Nomad v0.10 hosts work with v0.9 hosts. Upgrading 2 point releases (eg v0.8
+   to v0.10) may work but is untested and unsupported.
+
+   * Nomad does *not* support downgrading at this time.
```

Comment:
I would suggest expanding the sentence more - maybe something like "To downgrade Nomad, you will need to wipe its state and lose information about all jobs/nodes"